BUILD.gn

```diff
@@ -476,6 +497,13 @@ action("generate_upstream_blink_version") {
   ]
 }
 
+if (is_linux) {
+  pkg_config("libnotify_config") {
```

Comment:
Yes It is much cleaner so I moved the libnotify into separate file.
I checked how chrome use the `libgtkui`, (line 329-335 of chrome/browser/ui/BUILD.gn), so I didn't delete the `if(is_linux)`.